integrations/coinbase.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Coinbase",
+  "auth": {
+    "authorizationURL": "https://www.coinbase.com/oauth/authorize",
+    "tokenURL": "http://www.coinbase.com/oauth/token",
+    "accessTokenURL": "https://api.coinbase.com/oauth/token",
+    "authType": "OAUTH2",
+    "tokenParams": { "grant_type": "authorization_code" },
+    "authorizationParams": { "response_type": "code" }
+  },
+  "request": {
+    "baseURL": "https://api.coinbase.com/v2/user",
+    "headers": {
+      "Accept": "application/json",
+      "Authorization": "token ${auth.accessToken}",
```

Are you sure about the `Authorization` header? **token** doesn't look standard.

---

valid request token received is sent while making the api call

---

As per the doc, it should be 
```suggestion
      "Authorization": "Bearer ${auth.accessToken}",
```

See: https://developers.coinbase.com/docs/wallet/coinbase-connect/integrating (Step 4)

---

yay got it it whould be token type, instead of extracting we can have it default. thanks for feedback. will make the changes